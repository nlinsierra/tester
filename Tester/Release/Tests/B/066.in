8S TD TS 6D 9C KC JC 3D 9H KD JD 7S QC AC TC AD 6C 3H JH AS KS QH 5D TH 4H 8C 2D 4D AH 8D QD 9S 3C 3S 5S 9D 8H 7D KH 5H 6S QS 2H 4C 5C 7H 2S 4S JS
