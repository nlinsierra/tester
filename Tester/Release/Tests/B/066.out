3
2C 7C 6H
